src/views/StatusBoard.vue

```diff
@@ -2,9 +2,9 @@
   <div>
     <v-row class="pb-2" no-gutters>
       <v-col cols="12" md="9" class="d-flex align-baseline">
-        <h1 id="page-title">Evaluation Status Dashboard - Spring 2022</h1>
+        <h1 id="page-title">Evaluation Status Dashboard - {{ `${currentTermName}` }}</h1>
```

Comment:
@cesarvh - You don't need the back-ticks above. It can simplified to `{{ currentTermName }}`